apartmenttherapy/oxygen-lib-ui
# Shannon/quantcast

#### What does this PR do & why?
- change jwplayer events to run multiple functions on trigger

#### How can this be manually tested?
- please see pr: https://github.com/apartmenttherapy/oxygen/pull/669 for testing
- also run lib-ui tests

#### Any questions or concerns about this change?

_(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)_

#### Deployment Notes

_(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)_

#### Link to story card and other documentation
https://app.asana.com/0/543787180217047/791966078829326

#### GIF

#### Have you remembered to...?
